_shared/project/.github/workflows/ci.yml

```diff
@@ -26,10 +26,6 @@ jobs:
         {{- include(".github/workflows/ci/services.yml", indent=6) -}}
       {% endif %}
     {% endif %}
-    {% if job in ["tests", "functests"] and cookiecutter.get("postgres") == "yes" %}
-    env:
-      TEST_DATABASE_URL: postgresql://postgres@localhost:{{ cookiecutter['__postgres_port'] }}/{{ cookiecutter.package_name }}_test
-    {% endif %}
```

Comment:
This is no longer needed here because it's in `tox.ini` (and `ci.yml` runs the tests via tox).